src/store/scores.js

```diff
@@ -1,5 +1,303 @@
+      return(gradeNum.toString());
+    } else {
+      return("adult");
+    }
+
+  }
+
+};
+
+export function thetaToSupportSWR (percentile, grade) {
+  let support;
+
+  // we report automaticity instead of support for grades K/1 
+  if ((grade == "K") || (grade == "1")) {
+    support = (percentile < 50) ? "Limited" : "Average or Above Average";
+  } else {
+    support = (percentile < 25) ? "Extra Support Needed" : (percentile < 50) ? "Some Support Needed": "Average or Above Average";
+  }
+  return support;
+};
+
+export function woodcockJohnsonLookup (thetaEstimate) {
+  // TODO_Adam replace this totally fake calculation with a real lookup table based on thetaEstimate and ageMonths
+  console.log("WARNING: fake woodcockJohnsonLookup still in use");
+  return Math.round(100 * (thetaEstimate +4)/8);
+};
+
+export function percentileToSupportClassification(taskId, percentile, grade=1) {
+  let support = "";
+
+  switch(taskId) {
+    case "pa":
+      if ((grade == "K") || (grade <= "4")) {
+        support = (percentile < 25) ? "Extra Support Needed" : (percentile < 50) ? "Some Support Needed": "Average or Above Average";
+      } else {
+        support = (percentile < 15) ? "Extra Support Needed" : (percentile < 30) ? "Some Support Needed": "Average or Above Average";
+      }
+      break;
+
+    case "swr":
+    // we report automaticity instead of support for grades K/1 
+    if ((grade == "K") || (grade == "1")) {
+      support = (percentile < 50) ? "Limited" : "Average or Above Average";
+    } else {
+      support = (percentile < 25) ? "Extra Support Needed" : (percentile < 50) ? "Some Support Needed": "Average or Above Average";
+    }
+      break;
+
+    case "sre":
+    case "vocab":
+      console.log("TODO add sre and vocab cases to percentileToSupportClassification() ")
+      break;
+
+    default:
+      console.log(taskId, "missing from switch statement");
+  }
+
+  return(support);
+};
+
+export function countItems(dataArray, searchValue) {
+/*   //TODO_Adam -- how do I make the first param be treated as an array?
+  let count = dataArray.reduce((n, x) => n + (x === searchValue), 0);
+  return(count); */
+  return 0;  // TODO temp 
+};
+
+const gradeComparator = (a, b) => {
+  const order = ['pk', 'jk', 'tk', 'k', '1', '2', '3','4', '5', '6', '7', 
+                 '8', '9', '10', '11', '12', 'adult'];  
+
+  if (a === b) {
+    // equal inputs
+    return 0;
+  }
+
+  let indexA = order.length;
+  let indexB = order.length;
+
+  if (order.includes(a)) {
+    indexA = order.indexOf(a);
+  }
+
+  if (order.includes(b)) {
+    indexB = order.indexOf(b);
+  }
+
+  if (indexA === indexB) {
+    // neither input found
+    if (a > b)
+      return 1;
+    else
+      return -1;
+  } else if (indexA > indexB){
+    // string a was later in the array
+    return 1;
+  } else {
+    // b was later in the array
+    return -1;
+  }
+}
+
+export function debugTestFunction () {
```

Comment:
There's a few things that can be addressed later in this file, but is this function needed at all? It seems like it could be removed.